### ISSUE TITLE
Improve readme

### ISSUE THREAD

**Issue by phungvansyhb** [calumk/editorjs-codeflask#11]
Awesome library, but i have installed and try to know how to add language to config for tool :) because nothing in readme mention this

**Comment by sandro-pasquali** [comment]
Yes, it isn't clear how I can for example set the default language. Adding it in the normal config way doesn't seem to work, e.g.:

```
code: {
    class: editorjsCodeflask,
    config: {
        language: 'javascript'
    }
},
```
or:

```
code: {
    class: editorjsCodeflask,
    language: 'javascript'
},
```
Great tool! Being able to set a default language would be great.
